dockerfiles/Dockerfile.openvino

```diff
@@ -57,9 +56,13 @@ WORKDIR /code
 ENV PATH /opt/miniconda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:$PATH
 
 RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO /onnxruntime && \
-    /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
+    /bin/sh /onnxruntime/dockerfiles/scripts/install_common_deps.sh && \
     cd /onnxruntime/cmake/external/onnx && python3 setup.py install && \
     cp /onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt && \
+    cp /onnxruntime/dockerfiles/EULA.txt /code/EULA.txt && \
+    cp /onnxruntime/dockerfiles/ISSL.txt /code/ISSL.txt && \
+    cp /onnxruntime/dockerfiles/redist.txt /code/redist.txt && \
```

Comment:
/opt/intel/openvino_2019.1.144/licensing/deployment_tools/redist.txt
/opt/intel/openvino_2019.1.144/licensing/openvx/redist.txt
/opt/intel/openvino_2019.1.144/licensing/opencv/redist.txt
/opt/intel/openvino_2019.1.144/licensing/install_dependencies/redist.txt

Reply:
my understanding is we should remove these newly added files (EULA.txt, ISSL.txt, redist.txt) from the commit, and remove the cp command lines relate to them as well.

---
In reply to: [327868579](https://github.com/microsoft/onnxruntime/pull/1892#discussion_r327868579) [](ancestors = 327868579)